user/enterprise/build-images.md

```diff
@@ -75,85 +75,38 @@ To build docker images on Travis CI Enterprise, add the following to `/etc/defau
-```
-
-After that, both Docker and travis-worker have to be restarted. For Docker, please run:
+We accomplish this by adding another configuration option to `/etc/default/travis-worker`:
 
 ```
-$ sudo restart docker
+export TRAVIS_WORKER_DOCKER_BINDS='/var/run/docker.sock:/var/run/docker.sock'
 ```
 
-__Ubuntu 16.04 as Host operating system__
-
-To have the Docker daemon listening to the private IP address, edit `/etc/docker/daemon.json` and add the machine's private IP address incl. the Docker API port (in this example it's `172.31.7.199:4232`):
-
-```json
-{
-    //...
-    "hosts": [
-        "fd://",
-        "tcp://172.31.7.199:4243"
-    ]
-}
-```
-
-By default, the Docker daemon is configured to be accessible by  [socket activation](http://0pointer.de/blog/projects/socket-activation.html) (that's what `fd://` is for). To prevent issues related to having host configured in both service file and `/etc/docker/daemon.json` edit the Docker settings by by running `sudo systemctl edit docker`. It opens an editor with a new file where you can override Docker default settings. Please add the following in there:
-
-```
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd
-```
-
-You'll notice that in the above code snippet `ExecStart` gets assigned twice: first time with an empty value and the second time with a new command. This is a specific systemd semantic we need to follow here in order to customize the Docker start command.
-
-After that, issue an `sudo systemctl daemon-reload`, together with a `sudo systemctl restart docker`.
+With this option we tell `travis-worker` to make the host's Docker socket available inside the build containers. Please restart travis-worker after you have saved the configuration file.
 
 
 __Restart travis-worker__
 
 To restart travis-worker, you can find the instructions [here](https://docs.travis-ci.com/user/enterprise/worker-cli-commands/#Stopping-and-Starting-the-Worker).
 
-
 ### Updates to your .travis.yml files
 
 #### Trusty build containers
 
-> Please note: This solution utilizes the Docker daemon from the host machine - any containers and images you pull and run through your build need to be cleaned up manually.
-
-Add the following to your `.travis.yml` configuration:
-
-```
-install:
-  - sudo apt-get update
-  - sudo apt-get install -y curl software-properties-common apt-transport-https ca-certificates
-  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu trusty stable"
-  - sudo apt-get update
-  - sudo apt-get install -y docker-ce
-  - sudo docker -H <IP>:4243 pull ubuntu
-script:
-  - sudo docker -H <IP>:4243 run ubuntu date
+Once the worker machine is configured properly, you can use Docker as usual in your build. Please note that you don't need to add `services: docker` to the `.travis.yml`.
```

Comment:
Maybe linkify "configured properly" to the config instructions?
Also maybe worth a brief explanation to distinguish from the behavior on hosted? ("Please note that in an enterprise installation you don't need to add...")